modules/Charts/Dashlets/OpportunitiesByLeadSourceDashlet/OpportunitiesByLeadSourceDashlet.php

```diff
@@ -209,6 +209,9 @@ function getChartData($query)
         $row = $db->fetchByAssoc($result);
 
         while ($row != null){
+			if(isset($row['lead_source'])){
```

Comment:
Can you also please check if the $app_list_strings['lead_source_dom'][$row['lead_source']] is set. We don't want to delete a label value when it's may empty for some reason..